README.md

```diff
@@ -1,2 +1,220 @@
-# AirPollutionApp
-This is the capstone project for react module that used for finding current air pollution data using geographical coordinates (latitude, longitude).
+# Air-Pollution-App
+React group project
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <h3><b>Air Pollution App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Air-Pollution-App] <a name="about-project"></a>
+
+**[Air-Pollution-App]** "Air-Pollution-App" This is the capstone project for react module that used for finding current air pollution data using geographical coordinates (latitude, longitude). The application will allow users to:
+- Find the country,
+- See the current air pollution gases in this country.
+
+## 🛠 Built With 
+<a name="built-with"> React </a>
+<a name="built-with"> Redux </a>
+<a name="built-with"> React Bootstrap</a>
+<a name="built-with"> Hooks</a>
+
+### Tech Stack 
+<a name="tech-stack">React Redux</a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.org/">React</a></li>
+    <li><a href="https://css.org/">Redux</a></li>
+    <li><a href="https://css.org/">React Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Render</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://restcountries.com/">Restcountries API</a></li>
+   <li><a href="https://openweathermap.org/api/air-pollution">OpenWeatherMap API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[book_rockets]**
+- **[cancel_book_rocket]**
+- **[join_mission]**
+- **[leave_mission]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Screenshots
+
+## Countries_View
+![Countries_View](./src/assets/continents-screenshoot.png)
+
+## Pollution_View
+![Pollution_View](./src/assets/Pollution-screenshoot.png)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://)
```

Comment:
- [ ] Please make sure that the project is deployed, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=the%20design%20provided.-,Deploy%20the%20project,-and%20test%20for) requirement.

Reply:
Done 👌